src/test/java/com/github/cameltooling/lsp/internal/completion/CamelComponentOptionsCompletionsTest.java

```diff
@@ -69,7 +69,7 @@ public void testProvideCamelOptionsForConsumerOrProducer() throws Exception {
     	completionItem.setInsertText("clientConfigOptions=");
     	completionItem.setTextEdit(new TextEdit(new Range(new Position(0, 23), new Position(0, 23)), "clientConfigOptions="));
     	completionItem.setDocumentation("To configure the AsyncHttpClientConfig using the key/values from the Map.");
-    	completionItem.setDetail("java.util.Map<java.lang.String,java.lang.Object>");
+    	completionItem.setDetail("java.util.Map<java.lang.String, java.lang.Object>");
```

Comment:
Is the space really necessary?

Reply:
yes, it is reflecting what is provided by the Camel catalog